### ISSUE TITLE
<Model onLoadProgress={...} /> doesn't work

### ISSUE THREAD

**Issue by saidmoya12** [brianzinn/react-babylonjs#122]
onLoadProgress doesn't work

```jsx
<Model 
  onLoadProgress={(evt)=>{
   console.log(evt) //trigger no work at current version
}}/>
```

I'm testing with gltf model but doesn't work after update

**Comment by brianzinn** [comment]
You are right.  Will be available later today.

**Comment by brianzinn** [comment]
Thanks @saidmoya12 for reporting fixed in 3.0.10 (as well meshnames and error reporting)
